webui/src/main/java/org/polypheny/db/webui/Crud.java

```diff
@@ -2813,6 +2814,15 @@ void getDirectory( File dir, Context ctx ) {
         ctx.result( "" );
     }
 
+
+    void getCatalog( final Context ctx ) {
+        if ( Catalog.mode == RunMode.PRODUCTION ) {
```

Comment:
I would feel better if it would be the other way around, checking for DEVELOPMENT and forbidding otherwise.

Reply:
Then add both. Better than the risk of adding another production run-mode and forgetting to update this.